### ISSUE TITLE
remote.api: admin login

### ISSUE THREAD

**Issue by rjeczalik** [koding/koding#9667]
Kloud is going to be able to make requests to remote.api on behalf of the users.

It should be possible for kloud to use some persistent jSession.clientId and set the username in the payload / header.

Open for discussion.

**Comment by rjeczalik** [comment]
Blocked by https://github.com/koding/koding/pull/9548.

**Comment by gokmen** [comment]
do we need this for current implementation? or for the future use? for now kloud can work as is for kd.io requirements right?

**Comment by rjeczalik** [comment]
@gokmen How otherwise kloud will use remote.api to create resources (stack, machines etc.) for users?

I can continue with implementing mongo backend in kloud, then this issue can be solved at some later time.

**Comment by gokmen** [comment]
@rjeczalik main reason for implementing the api endpoints was developing kd.io without providing another backend for it. but of course we can use the same api for kloud operations as well. so, ignore my question. 

I can provide an endpoint for kloud to create a session on user behalf with provided username/password/team_name from them. and kloud can store this special session token in user home folder or in kite key somewhere which then make the whole requests with that. but you need to keep sessionTokens for each team separately since we're getting context (active team) of user from that session data. wdyt?

**Comment by rjeczalik** [comment]
> and kloud can store this special session token in user home folder or in kite key somewhere which then make the whole requests with that

If user needs to separately login to remote.api, I can just make `kd auth login` to login to remote.api and store clientId instead - and send clientId with every request to kloud. Thus no change would be required on remote.api part. I tried to be lazy.

**Comment by gokmen** [comment]
I think this is enough for the main operations, if you send clientId with every request from kloud to remote.api it will work as is.

**Comment by rjeczalik** [comment]
Not needed for now.
